NSS-Likeable-Lemurs/WorkforceManagement
# Hc department ticket6

## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/WorkforceManagement/issues/6)

## Description of Proposed Changes
- Created model for Department
- Created MVC for Department
-Created a view for Department
-Created Integration Tests for Creating a Model 
-Created Integration Tests for Posting a Model

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <HC-Department-Ticket6>

To Test: Run the Code - click on the Department Link in Navbar. Click on Create New Department, then Create a new Department and new Department Created should post to the page. The list of Departments should display with their respective budgets and number of employees.

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* DepartmentController, Department View, Department View Model, and Department Model

## Mentions @username

@Akeen11 
@kellycook301 
@priyankagargc27 
@streatorward 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Each method is commented with Author, Purpose, and Methods
   
2 The Department link on the Navbar - when pressed shows the Department's Name, Budget and Number of employees.
3. When user hits Create Department Link, should be able to write out the name of the Department and Budget.
4. When the user hits Save, then the new department created should post to the page
Every line of code has been peer reviewed.
5. Integration Testing - The Post Test and Get Test should run and work.
